sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -1000,7 +1000,22 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief Optional conditions that must be met to perform this operation.
      */
-    AppendBlobAccessConditions AccessConditions;
+    struct : public Azure::ModifiedConditions,
+             public Azure::MatchConditions,
+             public LeaseAccessConditions
+    {
+      /**
+       * @brief Ensures that the AppendBlock operation succeeds only if the append blob's size
+       * is less than or equal to this value.
+       */
+      Azure::Nullable<int64_t> IfMaxSizeLessThanOrEqual;
+
+      /**
+       * @brief Ensures that the AppendBlock operation succeeds only if the append position is equal
+       * to this value.
+       */
+      Azure::Nullable<int64_t> IfAppendPositionEqual;
+    } AccessConditions;
```

Comment:
Breaking change

Reply:
Do we still have this breaking change?

Reply:
No, no breaking changes will be made in this release.